internal/adapters/scheduler_storage/pg/scheduler_test.go

```diff
@@ -27,14 +27,35 @@ package pg
 
 import (
 	"testing"
+	"time"
+
+	"github.com/topfreegames/maestro/internal/core/entities/forwarder"
+
+	"github.com/topfreegames/maestro/internal/validations"
 
 	"github.com/stretchr/testify/require"
 	"github.com/topfreegames/maestro/internal/core/entities"
 	"github.com/topfreegames/maestro/internal/core/entities/game_room"
 )
 
 func TestScheduler_ToScheduler(t *testing.T) {
+	err := validations.RegisterValidations()
+	if err != nil {
+		t.Errorf("unexpected error %d'", err)
+	}
 	t.Run("valid schedulers", func(t *testing.T) {
+		fwd := &forwarder.Forwarder{
+			Name:    "fwd",
+			Enabled: true,
+			FwdType: forwarder.TypeGrpc,
+			Address: "address",
```

Comment:
@lyracampos @GuilhermBrSp This line is different from the RFC, but I think it makes a lot more sense (as we discussed before). WDYT?

Reply:
@caiordjesus are you talking about the name of FwdOptions property? 
i think you should use **Options** only, because you are already in forwarder package context. 

but if you don't want to use only Options, I suggest you to use **ForwarderOptions** instead, because Fwd doesn't means anything.